### ISSUE TITLE
Enhancement: Jump to next highlighted keyword

### ISSUE THREAD

**Issue by alex-trueman** [seanliang/HighlightWords#23]
A key-map to jump to the next highlighted keyword, much the same as find-next.

**Comment by seanliang** [comment]
Any keybinding suggestion, for all three platforms?

**Comment by alex-trueman** [comment]
Not really. It's tough to come up with something that doesn't conflict with other packages. Ctrl-Alt-F3 in Windows might work. It is like find next (F3), but might be too similar to find previous (Shift-F3) and find all (Alt-F3). I only use Windows so not to sure about other OSs.

**Comment by ghost** [comment]
Maybe just a new setting to automatically set bookmarks on each highlighted line (through ST's bookmark API). Then we would just use the already defined key bindings for next/previous bookmark.

**Comment by alex-trueman** [comment]
That would work well.
